### PR TITLE
[KYUUBI #2553] Kyuubi Spark TPC-DS Connector - Add tiny scale

### LINKED ISSUE #2553: [Subtask] Kyuubi Spark TPC-DS Connector - Add tiny scale
### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/incubator-kyuubi/issues) and found no similar issues.

### Describe the subtask

Current available scales: 0, 1, 10, 100, 300, 1000, 3000, 10000, 30000, 100000.

Event scale 1G is a little bit large for demo on laptop, we can introduce a scale 0.01 and name the database as `tiny` for demo purpose.

### Parent issue

https://github.com/apache/incubator-kyuubi/issues/2538

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by cxzl25**
### _Why are the changes needed?_

Current available scales: 0, 1, 10, 100, 300, 1000, 3000, 10000, 30000, 100000.

Event scale 1G is a little bit large for demo on laptop, we can introduce a scale 0.01 and name the database as `tiny` for demo purpose.
At the same time, it can also shorten the test time of UT.

close #2553

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by cxzl25** [comment]
```
./build/mvn test -Pspark-3.2 -Dtest=none -DwildcardSuites=org.apache.kyuubi.spark.connector.tpcds.TPCDSCatalogSuite,org.apache.kyuubi.spark.connector.tpcds.TPCDSQuerySuite,org.apache.kyuubi.spark.connector.tpcds.TPCDSTableSuite -pl 'extensions/spark/kyuubi-spark-connector-tpcds' -am
```

## Current

image

## Tiny
image

File: extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSStatisticsUtils.scala [hunk 1]
```diff
@@ -24,13 +24,18 @@ import io.trino.tpcds.Table._
 // Page 42 Table 3-2 Database Row Counts
 object TPCDSStatisticsUtils {
 
+  val TINY_SCHEMA_NAME = "tiny"
+
+  val TINY_SCALE_FACTOR = 0.01
```
Comment (pan3793) [inline extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSStatisticsUtils.scala:29]: Why not just merge TINY_SCALE_FACTOR into SCALES?

File: extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSStatisticsUtils.scala [hunk 2]
```diff
@@ -24,13 +24,18 @@ import io.trino.tpcds.Table._
...
+
   val SCALES: Array[Int] = Array(0, 1, 10, 100, 300, 1000, 3000, 10000, 30000, 100000)
 
   // https://github.com/Teradata/tpcds/issues/26
-  def numRows(table: Table, scale: Int): Long = {
-    require(SCALES.contains(scale), s"Unsupported scale $scale")
+  def numRows(table: Table, scale: Double): Long = {
+    require(SCALES.contains(scale) || scale == TINY_SCALE_FACTOR, s"Unsupported scale $scale")
```
Comment (pan3793) [inline extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSStatisticsUtils.scala:35]: it's not a good practice to use `==` to compare two double values, we should always use `abs(d1 - d2) < epsilon` for that.

File: extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSSchemaUtils.scala [hunk 3]
```diff
@@ -55,6 +55,14 @@ object TPCDSSchemaUtils {
       table.getGeneratorColumns.head.getGlobalColumnNumber
   }
 
+  def getDBNameByScale(scale: Double): String = {
```
Comment (pan3793) [inline extensions/spark/kyuubi-spark-connector-tpcds/src/main/scala/org/apache/kyuubi/spark/connector/tpcds/TPCDSSchemaUtils.scala:58]: getDBNameByScale => dbName

**Comment by pan3793** [comment]
Close in favor #2774

**Pull request closed without merging**